source/reference/minio-mc-admin/mc-admin-user-svcacct-list.rst

```diff
@@ -0,0 +1,82 @@
+.. _minio-mc-admin-svcacct-list:
+
+==============================
+``mc admin user svcacct list``
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin user svcacct list
+
+
+Syntax
+------
+
+.. start-mc-admin-svcacct-list-desc
+
+The :mc-cmd:`mc admin user svcacct list` command lists all access keys associated to the specified user.
+
+.. end-mc-admin-svcacct-list-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command lists all access keys associated to an existing MinIO user:  
```

Comment:
> ... associated to the ``admin1`` user on the ``myminio`` deployment.

This suggests replacing `myuser` with a different example, like `admin1` to avoid repetition of `user`.
You'd need to update the command on line 34 if you switch out `myuser`.

Reply:
I rewrote this earlier, but I like your `admin1` idea better.